partner_financial_risk/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage the partner's *Financial Risk*
+If any limit is exceed the the partner is blocked to confirm sale orders.
```

Comment:
I agrree with @elicoidal. If a module is not well explained, it cannot be well evaluated and used.